frontend/app/mocks/client-application-data/client-application.json

```diff
@@ -214,28 +215,9 @@
                 ]
               }
             ],
-            "LivingIndependentlyIndicator": "",
-            "PrivateDentalInsuranceIndicator": false,
-            "ProvincialDentalCoverageIndicator": ""
-          },
-          "PersonSINIdentification": {
-            "IdentificationID": "",
-            "IdentificationCategoryText": ""
-          },
-          "ClientIdentification": [
-            {
-              "IdentificationID": "",
-              "IdentificationCategoryText": ""
-            },
-            {
-              "IdentificationID": "",
-              "IdentificationCategoryText": ""
-            },
-            {
-              "IdentificationID": "",
-              "IdentificationCategoryText": ""
-            }
-          ]
```

Comment:
My mistake - I know I mentioned in our chat that the client number should be mapped to `IdentificationID` when `person.PersonSINIdentification.IdentificationCategoryText === ('Client Number')`, but it should be mapped to `person.ClientIdentification.IdentificationCategoryText === ('Client Number')`.

Reply:
Added back in the "ClientIdentification" section